tracee-ebpf/tracee/tracee.go

```diff
@@ -920,8 +923,9 @@ func (t *Tracee) getProcessCtx(hostTid int) (ProcessCtx, error) {
 		if err != nil {
 			return processCtx, err
 		}
-		processCtx, err = t.ParseProcessContext(processCtxBpfMap)
-		t.processTree.processTreeMap[hostTid] = processCtx
+		cgroupId := t.containers.GetCgroupInfo(binary.LittleEndian.Uint64(processCtxBpfMap[8:16])).ContainerId
```

Comment:
why not getting the containerID in the `processContext.ParseProcessContext()` func?
also - please change `cgroupId` var to `containerId` or something alike.

Reply:
Because of that if we will want to fetch the `containerId` from the `processContext.ParseProcessContext()` we will need to use` Tracee Containers API` which will lead us to append this function as Tracee function and then in will make the `process context package`  be apart of Tracee.

I found out that to get he `containerID` as argument will free us to include that function as Tracee and make the code more compressed with Tracee functions